coffee-mode.el

```diff
@@ -1197,6 +1197,38 @@ comments such as the following:
      (0 (ignore (coffee-syntax-propertize-block-comment)))))
    (point) end))
 
+(defun coffee-get-comment-info ()
+  (let* ((syntax (syntax-ppss))
+         (commentp (nth 4 syntax))
+         (comment-start (nth 8 syntax)))
+    (when commentp
+      (save-excursion
+        (if (string=
+             "###" (buffer-substring (- comment-start 2) (1+ comment-start)))
+            (list :type 'multiple-line :start (- comment-start 2))
+          (list :type 'single-line :start comment-start))))))
```

Comment:
`:start` parameter is not used anywhere. Is it enough to return only comment type ?